src/internal/connector/exchange/service_iterators.go

```diff
@@ -51,6 +51,10 @@ func filterContainersAndFillCollections(
 		// deleted from this map, leaving only the deleted folders behind
 		tombstones = makeTombstones(dps)
 		category   = qp.Category
+
+		// Stop-gap: Track folders by LocationPath and if there's duplicates pick
+		// the one with the lexicographically larger ID.
```

Comment:
Curious, is this documented behavior that larger ID == more recently created?

Reply:
This isn't documented anywhere, just something we've noticed as we've gone along. We're not hoping to rely on it in the long run, this is more a temporary thing to unblock some folks until we have a better solution ready

Reply:
Gotcha! Better solution is moving to folder ids?